TestsCalculator/Pages/LoginPage.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace TestsCalculator.Pages
+{
+    public class LoginPage
+    {
+        private IWebDriver _driver;
+
+        public LoginPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public IWebElement LoginField
+        {
+            get
+            {
+                return _driver.FindElement(By.Id("login"));
+            }
+        }
+        public IWebElement InfoMessages
+        {
+            get
+            {
+                return _driver.FindElement(By.Id("errorMessage"));
+            }
+        }
+
+        public IWebElement PasswordField => _driver.FindElement(By.Id("password"));
+        public IWebElement LoginButton => _driver.FindElement(By.Id("loginBtn"));
+        public IWebElement RemindPassBtn => _driver.FindElement(By.Id("remindBtn"));
```

Comment:
You may add more code in 'getter', e.g. to wait for element is visible.

```suggestion
        public IWebElement RemindPassBtn
        {
        get
            {
                 By id = By.Id("remindBtn"); 
                 new WebDriverWait(driver, TimeSpan.FromSeconds(10))
                      .Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(id));
                 return _driver.FindElement(id);
            }
        }
```